neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+
+import click
+import six
+from bravado.exception import HTTPBadRequest, HTTPNotFound, HTTPUnprocessableEntity
+from mock import NonCallableMagicMock
+
+from neptune.api_exceptions import ExperimentLimitReached, \
+    ExperimentValidationError, ProjectNotFound
+from neptune.exceptions import STYLES
+from neptune.internal.backends.hosted_neptune_backend import HostedNeptuneBackend
+from neptune.internal.utils.http import extract_response_field
+from neptune.projects import Project
+from neptune.utils import with_api_exceptions_handler
+
+_logger = logging.getLogger(__name__)
+
+
+class AlphaIntegrationBackend(HostedNeptuneBackend):
```

Comment:
Right now we simply inherit from 'HostedNeptuneBackend', but it's likely that we'll create own implementation based on abstract 'Backend' in the future.